x-pack/packages/kbn-elastic-assistant/impl/data_anonymization_editor/index.test.tsx

```diff
@@ -44,6 +44,8 @@ describe('DataAnonymizationEditor', () => {
         <DataAnonymizationEditor
           selectedPromptContext={mockSelectedPromptContext}
           setSelectedPromptContexts={jest.fn()}
+          currentReplacements={{}}
+          isFlyoutMode={false}
```

Comment:
Nit, would love to see tests use a `defaultProps` object so that updates like these only need to happen in one spot 😅

Reply:
++, let's standardize going forward